core/src/foundations/ionic.vars.scss

```
@@ -939,6 +367,578 @@ $ion-primitives-pink-1100: var(--ion-primitives-pink-1100, #8b2248);
+$ion-text-info: var(--ion-text-info, $ion-semantics-info-900);
+$ion-text-info-rgb: var(--ion-text-info-rgb, 13, 75, 195);
+$ion-text-warning: var(--ion-text-warning, $ion-semantics-warning-1000);
+$ion-text-warning-rgb: var(--ion-text-warning-rgb, 112, 75, 2);
+$ion-text-success: var(--ion-text-success, $ion-semantics-success-900);
+$ion-text-success-rgb: var(--ion-text-success-rgb, 18, 111, 35);
+$ion-text-link-default: var(--ion-text-link-default, $ion-semantics-primary-base);
+$ion-text-link-default-rgb: var(--ion-text-link-default-rgb, 16, 92, 239);
+$ion-text-link-press: var(--ion-text-link-press, $ion-semantics-primary-800);
+$ion-text-link-press-rgb: var(--ion-text-link-press-rgb, 15, 84, 218);
+$ion-text-link-visited: var(--ion-text-link-visited, $ion-semantics-primary-900);
+$ion-text-link-visited-rgb: var(--ion-text-link-visited-rgb, 13, 75, 195);
+$ion-text-inverse: var(--ion-text-inverse, $ion-primitives-base-white);
+$ion-text-inverse-rgb: var(--ion-text-inverse-rgb, 255, 255, 255);
+$ion-text-select: var(--ion-text-select, $ion-semantics-primary-800);
+$ion-text-select-rgb: var(--ion-text-select-rgb, 15, 84, 218);
+$ion-text-extended-pumpkin: var(--ion-text-extended-pumpkin, $ion-primitives-pumpkin-900);
+$ion-text-extended-pumpkin-rgb: var(--ion-text-extended-pumpkin-rgb, 128, 48, 4);
+$ion-text-extended-orange: var(--ion-text-extended-orange, $ion-primitives-orange-900);
+$ion-text-extended-orange-rgb: var(--ion-text-extended-orange-rgb, 132, 77, 0);
+$ion-text-extended-lime: var(--ion-text-extended-lime, $ion-primitives-lime-1000);
+$ion-text-extended-lime-rgb: var(--ion-text-extended-lime-rgb, 62, 87, 9);
+$ion-text-extended-teal: var(--ion-text-extended-teal, $ion-primitives-teal-900);
+$ion-text-extended-teal-rgb: var(--ion-text-extended-teal-rgb, 0, 107, 95);
+$ion-text-extended-aqua: var(--ion-text-extended-aqua, $ion-primitives-aqua-900);
+$ion-text-extended-aqua-rgb: var(--ion-text-extended-aqua-rgb, 20, 109, 129);
+$ion-text-extended-indigo: var(--ion-text-extended-indigo, $ion-primitives-indigo-900);
+$ion-text-extended-indigo-rgb: var(--ion-text-extended-indigo-rgb, 53, 22, 174);
+$ion-text-extended-violet: var(--ion-text-extended-violet, $ion-primitives-violet-900);
+$ion-text-extended-violet-rgb: var(--ion-text-extended-violet-rgb, 101, 26, 197);
+$ion-text-extended-purple: var(--ion-text-extended-purple, $ion-primitives-purple-1000);
+$ion-text-extended-purple-rgb: var(--ion-text-extended-purple-rgb, 129, 42, 170);
+$ion-text-extended-magenta: var(--ion-text-extended-magenta, $ion-primitives-magenta-1100);
+$ion-text-extended-magenta-rgb: var(--ion-text-extended-magenta-rgb, 137, 43, 122);
+$ion-text-extended-pink: var(--ion-text-extended-pink, $ion-primitives-pink-1000);
+$ion-text-extended-pink-rgb: var(--ion-text-extended-pink-rgb, 170, 42, 88);
+$ion-icon-default: var(--ion-icon-default, $ion-primitives-neutral-1200);
+$ion-icon-default-rgb: var(--ion-icon-default-rgb, 36, 36, 36);
```

```suggestion
$ion-icon-default-rgb: var(--ion-icon-default-rgb, $ion-primitives-neutral-1200-rgb);
```

Should this fallback to `$ion-primitives-neutral-1200-rgb` (and the same for all other icon rgb vars)?